### DISCUSSION TITLE
What's the purpose of 'onCreatingRowChange' ?

### DISCUSSION THREAD

**Discussion by ctretyak** [KevinVandy/mantine-react-table#155] [Q&A] [answered]
I found only 3 mentions and none of them is calling it. It doesn't work?
I'd like to subscribe when table.setCreatingRow / setEditingRow is called. onEditingRowChange works perfectly, but onCreatingRowChange is never called

![image](https://github.com/KevinVandy/mantine-react-table/assets/4935982/58c97236-be15-4e9d-ad86-9d3444a5bcc3)

**Comment by KevinVandy** [comment, marked as answer]
yeah, you're right, was an oversight where this state is always internal. Will publish a fix for that
